pac4j-oauth/src/main/java/org/pac4j/oauth/profile/generic/GenericOAuth20Profile.java

```diff
@@ -0,0 +1,89 @@
+package org.pac4j.oauth.profile.generic;
+
+import org.pac4j.core.context.Pac4jConstants;
+import org.pac4j.core.profile.Gender;
+import org.pac4j.oauth.profile.OAuth20Profile;
+
+import java.util.Locale;
+
+/**
+ * <p>This is the user profile for the generic OAuth 2.0 client.</p>
+ * <p>It is returned by the {@link org.pac4j.oauth.client.GenericOAuth20Client}  </p>
+ *
+ * @author aherrick
+ * @since 1.9.2
+ */
+public class GenericOAuth20Profile extends OAuth20Profile {
+
+    private transient GenericAttributesDefinition attributes = new GenericAttributesDefinition();
```

Comment:
I have a setter method a few lines below that allows it to be changed. My thought process is that you can create a custom attributes definition and pass it in when creating the client. Ex:
`private Class MyAttributes extends GenericAttributesDefinition {
static {
FAMILY_NAME = "last_name"
}
public MyAttributes() {
super()
}
}`

This tells the client that when it retrieves the profile, the FAMILY_NAME can be retrieves using the key "last_name".